rules/do-you-encourage-daily-exercise-at-work/rule.md

```diff
@@ -1,22 +1,20 @@
 ---
 type: rule
-archivedreason: 
 title: Do you encourage daily exercise at work?
-guid: 7255f54e-219b-46cd-b1d4-210997e5c99c
 uri: do-you-encourage-daily-exercise-at-work
-created: 2016-11-07T06:40:46.0000000Z
 authors: []
```

Comment:
Hi @miichaelsmedley ,

- [ ] 1, Please add Uly and youself as authors ;)

Reply:
done.